Automattic/sensei
# Add npm command for adding changelog entry

## Proposed Changes
An easy productivity boost that adds an `npm` command for generating a change log entry, since I can never remember the long version. 🙂  Also organizes the scripts in alphabetical order.

## Testing Instructions


1. Execute `npm run changelog` and ensure it kickstarts the change log entry creation process.

## Pre-Merge Checklist
